### ISSUE TITLE
Fix compressed navigation issue

### ISSUE THREAD

**Issue by grzanka** [yaptide/ui#1417]
When left menu navigation is compressed, then navigation by keyboard seems not work properly. 
Selected buttons doesn't have a text over it to explain what it does:

BAD
![image](https://github.com/yaptide/ui/assets/7374733/aeb15bb4-4775-4769-9ea8-d3a9d7639a8b)

GOOD
![image](https://github.com/yaptide/ui/assets/7374733/e70c0862-1f26-46fd-be5c-0d87b22ca576)

**Comment by matuszsmig** [comment]
At this moment when navigation is compressed, navigation by keyboard works properly.
